app/controllers/api/v0/swagger/models/highlight.rb

```diff
@@ -114,6 +116,19 @@ module Api::V0::Swagger::Models::Highlight
     end
   end
 
+  add_properties(:HighlightUpdate) do
+    property :color do
+      key :type, :string
+      # remove the anchors because swagger-codegen always escapes them
+      key :pattern, ::Highlight::VALID_COLOR.inspect[1..-2]
+      key :description, 'The highlight color.'
```

Comment:
should this say something a bit more specific like 'The highlighted color to update to'?

Reply:
Changed to "The new highlight color."